kyo-core/shared/src/main/scala/kyo/Fiber.scala

```diff
@@ -49,7 +49,7 @@ object Fiber extends FiberPlatformSpecific:
       * @return
       *   A Fiber that completes successfully with the given value
       */
-    def success[E, A](v: A): Fiber[E, A] = result(Result.success(v))
+    def success[E, A](v: A)(using Frame): Fiber[E, A] = result(Result.success(v))
```

Comment:
Do you know why the macro was not failing expansion before?

Reply:
ah, it's actually unnecessary. I had added it with the `IO` suspension but later realized that it's not necessary because the returned values are effectively final